src/main/controllers/SubscriptionConfigListLanguageController.ts

```diff
@@ -0,0 +1,27 @@
+import { PipRequest } from '../models/request/PipRequest';
+import { Response } from 'express';
+import { cloneDeep } from 'lodash';
+import { SubscriptionService } from '../service/SubscriptionService';
+
+const subscriptionsService = new SubscriptionService();
+
+export default class SubscriptionConfigListLanguageController {
+    public async get(req: PipRequest, res: Response): Promise<void> {
+        const listTypeLanguageSubscriptions = await subscriptionsService.getUserSubscriptionListLanguage(
+            req.user['userId']
+        );
+        if (req.query.error === 'true') {
```

Comment:
This feels wrong to check for query in the 'GET' method. We should add a POST method too for this class

Reply:
fixed.